unit/util/irep.cpp

```diff
@@ -22,7 +22,7 @@ SCENARIO("irept_memory", "[core][utils][irept]")
       const std::size_t ref_count_size = 0;
 #endif
 
-#ifndef USE_STRING
+#ifndef USE_STD_STRING
```

Comment:
Make commit message title more precise.
"various things": there's only one thing in this commit.